graphql-server/migrations/metadata.yaml

```diff
@@ -0,0 +1,379 @@
+version: 2
```

Comment:
not sure why this got included -- perhaps i'm on a different version? or should this be gitignored?

Reply:
Looks like migration v2 config - part of 1.2.0 release - https://github.com/hasura/graphql-engine/releases/tag/v1.2.0

I'd like to get here asap, probably after this weekend to not disrupt peak activity.

See https://trello.com/c/2SRKKk1A/67-xs-upgrade-hasura-image-to-v120-https-githubcom-hasura-graphql-engine-releases-tag-v120 and https://trello.com/c/MRVL1bPt/68-s-update-migrations-from-v1-v2-after-120.

Weird that it generated for you - Are you using the docker image from docker-compose.yaml? 

IF so, we may need to pin `graphql-server/config.yaml` at 1.0

Reply:
For now - I'll remove this file. But we should version control it, when we are 2.0 in the future.

Reply:
ahh, i see. I did update the hasura-cli when it prompted me to do so -- didn't realize that would have tangible effects. will revert.

Reply:
fwiw, i think that's what would happen if someone installed hasura today via
```sh
curl -L https://github.com/hasura/graphql-engine/raw/master/cli/get.sh | bash
```

Reply:
Got it - will see if there is a workaround for that